eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -17,6 +17,7 @@ parameters:
   GHReviewersVariable: '' 
   GHTeamReviewersVariable: '' # externally set, as eng-common does not have the identity-resolver. Run as pre-step
   OnboardingBranch: ''
+  PRLabels: "auto-merge"
```

Comment:
These changes look unrelated.

Reply:
I will comment out this one. Just want to have a test on what happened if adding the non-exist label.